task_geo/data_sources/us_data/__main__.py

```diff
@@ -0,0 +1,9 @@
+from demog import download_us_data, process_us_data
+
+def main():
```

Comment:
This function should be named `us_census` and be available to import from top level module, that is:

```python
from task_geo.data_sources.us_census import us_census
```

You can do that by defining it on `__init__.py` ( which by the way is necessary to make your module importable) or importing it there.

Also, make it return a `pandas.DataFrame` which is the required specification in the [Data Sources](https://github.com/CoronaWhy/task-geo/blob/master/docs/data_sources.rst) documentation.

Then on __main__ you can simply import  and call it and then save as csv the returned file.